src/modules/user/repository/userRepositories.ts

```diff
@@ -0,0 +1,9 @@
+import Users from "../../../databases/models/users"
+import authRepositories from "../../auth/repository/authRepositories"
+
+const UpdateUserByAttributes = async (userId: number, status: string) =>{
```

Comment:
We already have the UpdateUserByAttributes function into e-commerce-ninjas-bn\src\modules\auth\repository\authRepositories.ts which is doing the same job.